client/src/js/analyses/components/Pathoscope/__tests__/Toolbar.test.js

```diff
@@ -4,7 +4,14 @@ import {
     TOGGLE_FILTER_ISOLATES,
     TOGGLE_FILTER_OTUS
 } from "../../../../app/actionTypes";
-import { mapDispatchToProps, mapStateToProps, PathoscopeDownloadDropdownTitle, PathoscopeToolbar } from "../Toolbar";
+import { Select } from "../../../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Select' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5211723)